### DISCUSSION TITLE
NocoDB meta tables when connecting to a new data source [Jul 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6044] [Discord Threads]
> 🙋‍♂️ **sami**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### If you are connecting to a new data source (RDS DB), only junction tables will be created as part of your connected data source instance. Junction tables are required to manage Links that you create (many-to-many).    
####    
#### Junction tables names will be of format : `nc_1ctb___nc_m2m_kx82dwol7p`   
####    
#### Can you provide list of any other tables that you see excluding m2m?  
#  
> 🙋‍♂️ **sami**  
#### These are all the tables I am seeing. And the section I removed are the tables that I am expecting to see. Currently I am viewing the database through plato.io but all these tables are visible when we access the DB through NocoDB as well  
![image0](https://cdn.discordapp.com/attachments/1115054360347545601/1115268617165082624/Screenshot_2023-06-05_at_14.14.17.png)  
![image1](https://cdn.discordapp.com/attachments/1115054360347545601/1115268617488040026/Screenshot_2023-06-05_at_14.14.23.png)  
![image2](https://cdn.discordapp.com/attachments/1115054360347545601/1115268617819410492/Screenshot_2023-06-05_at_14.14.28.png)  
#  
> 👨‍🦲 **dstala**  
#### these are nocodb meta columns & should be part of only meta DB. if you have connected to external data source, these will not be visible as part of that instance  
#  
> 👨‍🦲 **dstala**  
#### So, it looks like you have created tables using NocoDB.    
#### Refer here https://docs.nocodb.com/setup-and-usages/meta-management#accessing-data-sources  
#  
> 👨‍🦲 **dstala**  
#### When you open data-sources in settings, if you have connected to external source - you should see two sources listed.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1115054360347545601)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
